src/preprocess/links.rs

```diff
@@ -22,14 +24,38 @@ pub fn replace_all<P: AsRef<Path>>(s: &str, path: P) -> Result<String> {
     Ok(replaced)
 }
 
-#[derive(PartialOrd, PartialEq, Debug, Clone)]
```

Comment:
Note: Ranges are not implementing this. However, we are also not using it anywhere.

Reply:
Seems fine to me, `PartialOrd` isn't overly useful anyway.